### PR TITLE
house keeping + SDK upgrade

### PR DESCRIPTION
**Pull request by axe312ger**
Applies latest build size optimisation tricks to the bundle and upgrades the SDK to the current RC.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,6 +1,17 @@
 {
   "presets": [
-    "es2015",
+    ["env", {
+      "targets": {
+        "browsers": [
+          "last 2 versions",
+          "not ie < 13",
+          "not android < 50"
```
Comment (Khaledgarbaya) [inline .babelrc:8]: The Discovery app is meant to be Desktop only

**Review by Khaledgarbaya — COMMENTED** [review]
Looks good, 👏 
One small comment

File: .babelrc (hunk 1 above)
Comment (axe312ger) [reply .babelrc:8]: Browserlist does not offer any option to do desktop only. I already excluded old android browser, so the babel plugins are on a minimal already.

I am pretty sure it is fine like that :)

**Review by Khaledgarbaya — APPROVED** [review]
LGTM

**Pull request closed without merging**